resources/netlog_defs.ts

```diff
@@ -896,6 +896,28 @@ const latestLogDefinitions = {
     canAnonymize: false,
     firstOptionalField: undefined,
   },
+  MapEffect: {
+    type: '257',
+    name: 'MapEffect',
+    source: 'FFXIV_ACT_Plugin',
+    // in the 2.6.6.7 beta, ACT appears to post these messages with the int value instead of
+    // a string for messageType, but need to confirm once build is final.
+    messageType: '257',
```

Comment:
```suggestion
    messageType: '257',
```

I think this is correct.  These will not have a special parsed string like `'NetworkUpdateHP'`.